dizoo/box2d/car_racing/config/car_racing_ppo_config.py

```diff
@@ -0,0 +1,59 @@
+from easydict import EasyDict
```

Comment:
modify pull request name following this format ([link](https://di-engine-docs.readthedocs.io/zh_CN/latest/guide/warmup_zh.html#git-commit))